addons/sys_attenuate/XEH_preInit.sqf

```diff
@@ -10,4 +8,6 @@ PREP_RECOMPILE_END;
 
 GVAR(attenuationCache) = HASH_CREATE;
 
-ADDON = true;
+if (!hasInterface) exitWith {
```

Comment:
OK, This is the other way around. While this improves readability in shorter files, longer ones like sys_core (XEH_postInit), the solution taken here and started by snippers is more clear in my opinion.